test/mpi/runtests.in

```diff
@@ -769,9 +780,32 @@ sub RunMPIProgram {
     if ($test_opt->{init}) {
         $test_opt->{init}->();
     }
+
+    # acquire lock if requested
+    my $lockfile="/tmp/runtests-mem.lock";
+    my ($lock_sh, $lock_ex) = (1, 2);
```

Comment:
how is the regular lock intended to be used? i only see memory locking in this PR. is that right?

Reply:
It is enabled directly by setting `lock=name` in the testlines. Comment added.